source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -0,0 +1,830 @@
+|                       |                       |      Load Sample if   |
+|                       |                       |      the slot is      |
+|                       |                       |      empty.           |
++-----------------------+-----------------------+-----------------------+
+| Alternative           |    | (P1-4) Pitch     |    | Change Range     |
+|                       |      Play             |    | Hold and         |
+|                       |    | Play a single    |      selecting        |
+|                       |      cue Point at     |      SAMPLE, changes  |
+|                       |      variable         |      Range 3 ranges.  |
+|                       |      pitches.         |                       |
+|                       |                       |    | Change cue       |
+|                       |    | *Note*           |    | Hold and         |
+|                       |    | • Only One Cue   |      selecting other  |
+|                       |      at a time •      |      cue point on Cue |
+|                       |      Deleting cue     |      Pad changes cue. |
+|                       |      point in Pitch   |                       |
+|                       |      Play Mode not    |                       |
+|                       |      possible,        |                       |
++-----------------------+-----------------------+-----------------------+
+
+..
+
+**Advanced User Section**
+----------------------
+
+For advanced users, alternative functions exist in the code, by
+accessing and setting a code variable.
+
+**TAPPING**
+Added logic for the tap button, Two modes exist:
+
+The default is to use the MIXXX common bpm.tapButton
+which sets the effective bpm to the one tapped using the 
+tempo adjust. Shift tap resets to 0 tempo change. 
+
+The alternative changes the actual file bpm. The problem 
+is the reset does not change the file back to the original bpm, the best 
+I can do is change the effective bpm to the original, but then the file is 
+still on the changed bpm next time it is loaded. The alternative mode is only 
+accessible by setting a code variable MixtrackPlatinumFX.tapChangesTempo to 
+false, recommended only for advanced users.
+ 
+..
+
+For the default, the MIXXX common function takes taps and averages
+them. If you don't tap for 2 seconds the average resets and you start
+again. To prevent accidental double taps or misses if a tap is 40%
+shorter or 80% longer it will be ignored. I found while testing
+sometimes if I got the first two taps wrong the rest would be
+rejected by the filter, but it wasn't obvious this was happening. By
+default the button is dimly lit (like most others). When you tap the
+button if it accepts the tap it will go bright, if it rejects it from
+the filter it will show off. If this happens stop tapping wait 2
+seconds for the filter to clear and try again. Fixed this kicking in
+after 8 taps.
```

Comment:
Does this apply to your custom BPM tap function only?
If no, I don't think this information belongs here. Rather add that hint to https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[ChannelN]-bpm_tap (in a separate PR) and link to it with
```
:mixxx:coref:`bpm_tap<[ChannelN],bpm_tap>`
```

Reply:
Deleted